servers/physics_server_3d.cpp

```diff
@@ -231,10 +231,10 @@ void PhysicsShapeQueryParameters3D::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("set_exclude", "exclude"), &PhysicsShapeQueryParameters3D::set_exclude);
 	ClassDB::bind_method(D_METHOD("get_exclude"), &PhysicsShapeQueryParameters3D::get_exclude);
 
-	ClassDB::bind_method(D_METHOD("set_collide_with_bodies", "enable"), &PhysicsShapeQueryParameters3D::set_collide_with_bodies);
```

Comment:
In order to make things more explicit, you can use `set_colliding_with_bodies` (without the default value), `is_colliding_with_bodies`, `set_colliding_with_areas`, `is_colliding_with_areas`.

See #33026 for the discussion about boolean setters/getters convention.